packages/firestore/test/integration/api/query.test.ts

```diff
@@ -566,4 +567,124 @@ apiDescribe('Queries', persistence => {
       }
     }).to.throw(expectedError);
   });
+
+  it('support collection groups', async () => {
```

Comment:
When reading this as English I would expect this to read "it supports collection groups". Would it be reasonable to make these `it('supports ...', ...)`?